js/main.js

```diff
@@ -106,4 +106,74 @@ null = Object
 / divide
 
 // oder of operations
-4 + 1 * 2 * 4 + 2
\ No newline at end of file
+4 + 1 * 2 * 4 + 2
```

Comment:
Hi @Simpji  i think you should comment out these lines.  They aren't javascript code and would throw an error when you run the code